book/website/project-design/data-governance/data-gov-ml.md

```diff
@@ -0,0 +1,67 @@
+(pd-dg-ml)=
+
+# Data Governance for the Machine Learning Pipeline
+
+The Machine Learning (ML) pipeline consists of a series of activities including the collection of data, training of an ML model, and the deployment of the model into use. 
+Data is integral throughout the ML process and the methods for which data is collected, annotated, processed, and shared will impact individuals who may be the subjects in or creators of the data as well as communities represented in the data.
+
+This chapter will cover examples of data governance practices for ML for different steps in the pipeline such as: 
+- Data Collection
+- Data Management
+- Data Processing
+
+## Data Collection
+
+Many ML models are trained using datasets collected by the research team, which may be proprietary, or by using an open dataset that is available for download.
```

Comment:
```suggestion
Many ML models are trained using datasets collected by a research team, which may be proprietary, or by using an open dataset that is available for download (sometimes with restrictions on its usage).
```

Clarified since many models are only open for academic research.